### PR TITLE
_posts: add notice about pam_tally breakage in pending pam upgrade

### PR DESCRIPTION
**Pull request by ahesford**
When this is merged, I'll pin the same text on Reddit for maximum attention.

### CODE REVIEW

File: _posts/2020-12-11-pam-1.5.1.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Upcoming PAM upgrade requires review of custom configurations
+layout: post
+---
+
+Void Linux current packages PAM version 1.3.0 and, until today, a standard PAM
```
Comment (zdykstra) [inline _posts/2020-12-11-pam-1.5.1.markdown:6]: s/current/currently

File: _posts/2020-12-11-pam-1.5.1.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+configuration that requires the `pam_tally` authentication module. This module
```
Comment (ericonr) [inline _posts/2020-12-11-pam-1.5.1.markdown:7]: `pam_tally` and `pam_lastlog`

File: _posts/2020-12-11-pam-1.5.1.markdown [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+was [deprecated in 1.4.0 and removed in 1.5.0](https://raw.githubusercontent.com/linux-pam/linux-pam/master/NEWS).
+The Void team plans to [upgrade to PAM 1.5.1](https://github.com/void-linux/void-packages/pull/25506)
+by the end of the year. For the upgraded PAM to function properly, any
+`pam_tally` requirements must be removed from the configuration files in
+`/etc/pam.d`. An updated [`pam-base` package](https://github.com/void-linux/void-packages/commit/e065ac229d78c4a1819098c7bda45bbd717120ba)
+released today removes the `pam_tally` requirement from the standard
+configuration. If you have not modified the standard PAM configuration,
+upgrading the `pam-base` package should be sufficient to adjust your
+configuration.
```
Comment (ericonr) [inline _posts/2020-12-11-pam-1.5.1.markdown:16]: Could mention XBPS's `keepconf` ? With a link to xbps.d(5)

File: _posts/2020-12-11-pam-1.5.1.markdown (hunk 2 above)
Comment (ericonr) [reply _posts/2020-12-11-pam-1.5.1.markdown:7]: This goes for the whole paragraph.

**Review by ericonr — APPROVED** [review]

**Comment by hippi777** [comment]
hi there! :)

im not familiar with pam, but if somebody affected wouldnt recognize it, what would be the worst? if anything nontrivial, then the info could contain troubleshooting as well.

otherwise, i think a mechanism would be better, as that way users doesnt need to follow void news to avoid surprises, and it could be an installation error with a nice message. maybe it is even a thing to detect the existence of those strings to avoid false positives, so ppl just wont be able to upgrade in such case, and i think such a mechanism could be beneficial in the future as well, and i guess it can be solved at a point before things getting messed up :D

bests! :)

**Comment by ericonr** [comment]
We can make `pam` depend on a specific version of `pam-base`, at the very least.

**Pull request merged**